src/pages/python/adding-alerts-and-slos.mdx

```diff
@@ -0,0 +1,57 @@
+import { Callout, Steps, Tabs, Tab } from "nextra-theme-docs";
+import AlertsSLOsLangIntro from "@/snippets/adding-alerts-slos.mdx";
+
+<AlertsSLOsLangIntro language="Python"/>
+
+## Pre-requisites
+
+1. Make sure you have setup and initialized Autometrics libraries as described in [Quickstart section](quickstart).
+
+2. Add the pre-configured Prometheus rules file to your Prometheus configuration. This file is located in the [Autometrics shared repository](https://github.com/autometrics-dev/autometrics-shared/blob/main/autometrics.rules.yml)
+
+## Usage
+
+<Steps>
+
+### Create `Objectives` with your desired SLOs
+
+To use autometrics SLOs and alerts, create one or multiple Objectives based on the function(s) success rate and/or latency, as shown below. 
+
+```python
+from autometrics import autometrics
+from autometrics.objectives import Objective, ObjectiveLatency, ObjectivePercentile
+
+API_SLO = Objective(
+    "random",
```

Comment:
it's unclear to me what "random" means here, could be good to make a more descriptive name

Reply:
In the example app in the repo we also use this name for the objective as it is applied to the `random_error` function. @brettimus  Do you think your feedback also applies to `examples/example.py` file in the autometrics-py repo? Or is that less of an issue there. because the objective definition lives right above the function definition there?